frontend/calendar1.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Oklahoma Department of Human Services</title>
+
+        <!--viewport metas go here-->
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="base.css">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+
+    <body>
+        <header class="banner">
+            <img class="logo" src="" alt="DHS logo">
+            <a class="web_title" href="index.html">OKDHS Adult and Family Services Information Center</a>
+
+            <nav class="generalNav">
+                <ul>
+                    <li><a href="profile.html"><img src="" alt="profile">Profile</a></li>
+                    <li><a href="calendar1.html"><img src="" alt="calendar">Calendar</a></li>
+                    <li><a href=""><img src="" alt="upload">Uploads</a></li>
+                    <li><a href="help1.html"><img src="" alt="help">Help</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <article>
+            <h1>OKDHS AFS Events and Program Deadlines</h1>
+            
+            <section class="dealines">
+                <h2>Upcoming Deadlines</h2>
+                <table>
+                    <tr>
+                        <th>Name</th>
+                        <th>Dates</th> 
+                    </tr>
+                    <tr>
+                        <td>SNAP Updates</td>
+                        <td>4/05/2019</td>
+                    </tr>
+                    <tr>
+                        <td>LIHEAP Begins</td>
+                        <td>5/30/2019</td>
+                    </tr>
+                    <tr>
+                        <td>LIHEAP Ends</td>
+                        <td>7/29/2019</td>
+                    </tr>
+                </table>
+            </section>
+
+            <section class="calendar">
```

Comment:
Calendar should also be generated dynamically. We need to discuss how we want to visually put the data/etc onto the calendar.

Reply:
color dots or something